### ISSUE TITLE
Playback position window not tab accessible

### ISSUE THREAD

**Issue by DilsonsPickles** [audacity/audacity#11652]
### Description

Open a project and press either the [ or ] key to summon the Playback position window. Press tab to move tab focus to nested controls and observe how keyboard focus does not move to controls. User can press esc to close window, but cannot tab to press enter to confirm changes/edits.

**Comment by DavidBailes** [comment]
@DilsonsPickles 

See: https://github.com/audacity/audacity/issues/9385 and https://github.com/audacity/audacity/issues/9381
